aiida_lsmo/workchains/cp2k_binding_energy.py

```diff
@@ -1,9 +1,7 @@
 # -*- coding: utf-8 -*-
 """Binding energy workchain"""
 
-import os
 from copy import deepcopy
-import ruamel.yaml as yaml  # does not convert OFF to False
```

Comment:
Need to check later if we make sure that OFF commands for CP2K are not converted to False

Reply:
I was a bit puzzled by the comment in the code - if `'OFF'` is stored as a string, no yaml converter should touch this.
Do you have an example?

Reply:
I don't have an example, you have to check back to the commit when I introduced ruaml: back then pyyaml with the standard yaml loader was taking some actions on the logic of the conversion from yaml to dict, and one of these was converting "OFF" string to "False" boolean. It is not that surprising, also pythonic "None" becomes the sting "Null" in a yaml file.

Reply:
> It is not that surprising, also pythonic "None" becomes the sting "Null" in a yaml file.

`None` has a special meaning in python, which is best represented by `null` in yaml.
`'OFF'` does not, so it would be very surprising if a yaml converter decides to convert it to a boolean (the same holds if you were to provide `'None'` as a string - also this should not be touched by a yaml converter)

Reply:
Actually this is still the case!
```
In [4]: yaml.safe_load("""
   ...: pippo: OFF
   ...: """)
Out[4]: {'pippo': False}
```
(yaml v.5.1.2)

Reply:
Ok, I see now that `off` does have special meaning in yaml.
http://rigaux.org/language-study/syntax-across-languages-per-language/YAML.html

The simple solution to circumvent this would be to put it into quotation marks

Reply:
It seems you are currently using `ruamel` consistently:

```
$ git grep yaml | grep -i import
calcfunctions/ff_builder_module.py:import ruamel.yaml as yaml
workchains/cp2k_multistage_protocols/__init__.py:import ruamel.yaml as yaml  # does not convert OFF to False
workchains/isotherm.py:    import ruamel.yaml as yaml
workchains/multicomp_ads_des.py:import ruamel.yaml as yaml
workchains/multicomp_gcmc.py:import ruamel.yaml as yaml
workchains/sim_annealing.py:from aiida_lsmo.calcfunctions.ff_builder_module import load_yaml
```

If you want to be absolutely safe, then you can also put `'OFF'` instead of `OFF`.
I guess for the moment it's ok.

Reply:
> The simple solution to circumvent this would be to put it into quotation marks

This is what I did in the beginning, but it is extremely likely that at some point you (or a new users) forget to do it because you don't see the point of using quotation marks and it crashes! Better to stick with ruamel.